source/manual/rename-a-country.html.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Rename a country
+parent: "/manual.html"
+layout: manual_layout
+section: Publishing
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2018-06-19
+review_in: 6 months
+related_applications: [travel-advice-publisher, content-tagger, whitehall]
+---
+
+We recommend testing out these instructions in integration and then staging before using them in production.
+
+### 1. Update Foreign Travel Advice publisher
+
+This will update www.gov.uk/foreign-travel-advice/countryname. In [Travel Advice Publisher](https://github.com/alphagov/travel-advice-publisher):
+
+1. Create a pull request with:
+* A migration to update the relevant `country_slug` of TravelAdviceEdition. [Example](https://github.com/alphagov/travel-advice-publisher/pull/346/commits/b28ff7b4eae96543324f61be700dca32f1ffdba5)
+* A change of the relevant name and slug in the `lib/data/countries.yml` file. Keep the same `content_id` and `email_signup_content_id`, and ensure the alphabetical order of the list is respected. [Example](https://github.com/alphagov/travel-advice-publisher/pull/346/commits/3eb10a8519638850760698992dd1f6467b041ab0)
+
+2. Deploy Travel Advice publisher
+Once the above PRs are ready, deploy Travel Advice Publisher. This should be an `app:migrate_and_hard_restart` deploy as a hard restart is required to update the yml file.
+
+3. Run rake tasks
+* Run `bundle exec rake publishing_api:republish_edition[country_slug]` to update the PublishingApi.
```

Comment:
You could have links here to the job in jenkins, for example:

https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=publishing-api&MACHINE=publishing-api-1.backend&RAKE_TASK=publishing_api:republish_edition[country_slug]